### PR TITLE
Implements getPost Form and History 

### PR DESCRIPTION
**Pull request by cocovsack**
Adds a form page where users can input comments (along with name/email). The comment history is also saved/displayed.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -46,6 +43,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String comment = getParameter(request, "comment", "");
+    String name = getParameter(request, "fname", "");
+
```
Comment (liguanghua) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:51]: Nitpick: this empty lines (here and line 53, 55) are not necessary.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -46,6 +43,29 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     response.getWriter().println(json);
   }
 
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String comment = getParameter(request, "comment", "");
```
Comment (liguanghua) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:49]: Do you really want to add empty comments to your database?

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -13,19 +13,15 @@
...
+    const historyElement = document.getElementById('history');
 
-    messageElement.innerHTML = '';
-    messageElement.appendChild(
-        createListElement('Message: ' + messages[0]));
-    messageElement.appendChild(
-        createListElement('Message: ' + messages[1]));
-    messageElement.appendChild(
-        createListElement('Message: ' + messages[2]));
+    messages.forEach((line) => {
```
Comment (liguanghua) [inline portfolio/src/main/webapp/script.js:22]: This is even better then a for...of loop :D

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -13,19 +13,15 @@
 // limitations under the License.
 
 /**
- * Generates a greeting to the user from the DataServlet server and adds it to the DOM
+ * Gets past comment history from the DataServlet server and adds it to the DOM
  */
-function getGreeting() {
+function getHistory() {
```
Comment (linxiz) [inline portfolio/src/main/webapp/script.js:18]: nit: consider renaming to getComments or getCommentHistory instead

File: portfolio/src/main/webapp/comment.html [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+  </body>
+</html>
+
+
+
+
+
+      
+
+
```
Comment (linxiz) [inline portfolio/src/main/webapp/comment.html:52]: nit: remove the unused comments and the extra empty lines

**Pull request merged**